test/srt/openai/test_serving_embedding.py

```diff
@@ -24,7 +24,7 @@
     MultimodalEmbeddingInput,
     UsageInfo,
 )
-from sglang.srt.entrypoints.openai.serving_embedding import EmbeddingHandler
+from sglang.srt.entrypoints.openai.serving_embedding import OpenAIServingEmbedding
 from sglang.srt.managers.io_struct import EmbeddingReqInput
```

Comment:
Same, lets remove the following test classes:

`TestCompatibilityWithAdapter`
`TestEdgeCases`